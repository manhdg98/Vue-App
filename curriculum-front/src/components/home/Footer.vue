<template>
  <v-footer
    absolute
    class="font-weight-medium"
  >
    <v-col
      class="text-center"
      cols="12"
    >
      &copy; {{ new Date().toLocaleDateString() }} â€” <strong>Curricula App</strong>
    </v-col>
  </v-footer>  
</template>
