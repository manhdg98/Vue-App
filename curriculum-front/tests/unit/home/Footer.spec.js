import { mount } from '@vue/test-utils'
import Footer from '@/components/home/Footer.vue'

describe('Footer.vue', () => {
  test('if snapshot is correct', () => {
    const wrapper = mount(Footer)
    expect(wrapper).toMatchSnapshot()
  })
})
