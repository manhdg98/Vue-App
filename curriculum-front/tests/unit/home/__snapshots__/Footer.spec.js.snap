// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue if snapshot is correct 1`] = `
<footer class="v-footer font-weight-medium v-sheet v-sheet--tile theme--light v-footer--absolute" style="left: 0px; right: 0px; bottom: 0px;">
  <div class="text-center col col-12">
    © 2020 — <strong>Curricula App</strong></div>
</footer>
`;
